k8s-operator/apis/v1alpha1/types_proxyclass.go

```diff
@@ -166,15 +180,15 @@ type Container struct {
 	// https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
-	// List of environment variables to set in the container.
-	// https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables
-	// Note that environment variables provided here will take precedence
-	// over Tailscale-specific environment variables set by the operator,
-	// however running proxies with custom values for Tailscale environment
-	// variables (i.e TS_USERSPACE) is not recommended and might break in
-	// the future.
+	// Container security context.
+	// Security context specified here will override the security context by the operator.
+	// By default the operator:
+	// - sets 'privileged: true' for the init container
+	// - set NET_ADMIN capability for tailscale container for proxies that
+	// are created for Services or Connector.
+	// https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
 	// +optional
-	Env []Env `json:"env,omitempty"`
+	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
```

Comment:
This is just me shifting those fields around to order them alphabetically (just to make it easier to read the code)